game.cpp

```diff
@@ -1785,11 +1785,13 @@ void test_game() //!OCLINT tests may be many
   }
   #endif // FIX_ISSUE_241
 
-  //#define FIX_ISSUE_457
+
+ #define FIX_ISSUE_457
   #ifdef FIX_ISSUE_457
+
   {
     // (457) The color of any player can be accessed easily
-    const game g;
```

Comment:
I disagree with this: changing the test is no way to solve it. Unless of course, the test is wrong, but I don't see why that would be the case here. Obtaining the colour of a player shouldn't change anything about the game or anythign inside it.

Reply:
Thanks, Theo. I added `const` before color and Tianjian taught me how this works, I learned a lot. :)

Reply:
I took the freedom to put back the `const` in game in the test, it still passes 👍